tests/test_integration_lazy_samples.py

```diff
@@ -53,15 +54,13 @@ def run_training_test(root_dir, device="cuda:0", cachedataset=0, readers=(None,
             mt.RandCropByPosNegLabeld(
                 keys=["img", "seg"], label_key="seg", spatial_size=[76, 82, 80], pos=1, neg=1, num_samples=4
             ),
-            mt.RandRotate90d(keys=["img", "seg"], prob=0.8, spatial_axes=[0, 2]),
+            mt.RandRotate90d(keys=["img", "seg"], prob=0.8, spatial_axes=(0, 2)),
             mt.ResizeWithPadOrCropD(keys=["img", "seg"], spatial_size=[80, 72, 80]),
             mt.Rotated(keys=["img", "seg"], angle=[np.pi / 2, np.pi / 2, 0], mode="nearest", keep_size=False),
         ],
         lazy_evaluation=lazy,
-        mode=("bilinear", 0),
-        padding_mode=("border", "nearest"),
-        lazy_keys=("img", "seg"),
-        lazy_dtype=(torch.float32, torch.uint8),
+        overrides=lazy_kwargs,
+        override_keys=("img", "seg"),
```

Comment:
hi @atbenmurray am I using the new API correctly? couldn't pass the integration tests here.

Reply:
Taking a look

Reply:
thanks, I fixed it https://github.com/wyli/MONAI/pull/19#discussion_r1144645919